docs/lib/menu.json

```diff
@@ -1,6 +1,7 @@
 {
   "items": [
     "project-setup",
+    "debugging",
```

Comment:
should debugging be the first thing after "project-setup", I'd believe it should be probably the very bottom?